sentences/sv/binary_sensor_HassGetState.yaml

```diff
@@ -0,0 +1,278 @@
+language: "sv"
+intents:
+  HassGetState:
+    data:
+      # https://www.home-assistant.io/integrations/binary_sensor/
+      # Battery
+      - sentences:
+          - "är <name> [batteri[et]] {bs_battery_states:state} [<i_på> <area>q]"
+          - "är batteriet [till] <name> {bs_battery_states:state} [<i_på> <area>q]"
+        response: one_yesno
+        requires_context:
+          domain: binary_sensor
+          device_class: battery
+        slots:
+          domain: binary_sensor
+          device_class: battery
+
+      - sentences:
+          - "är (något|några) batteri[er] {bs_battery_states:state} [<i_på> <area>q]"
+        response: any
+        slots:
+          domain: binary_sensor
+          device_class: battery
+
+      - sentences:
+          - "är alla batteri[er] {bs_battery_states:state} [<i_på> <area>q]"
```

Comment:
"är alla batterierna normala i köket" <- Guess we could extract `(batteri|enhet)[er]` to an expansion_rules and add all different versions.